definitions/ext-oracle/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
```

Comment:
We notice an INFRA-ORACLEINSTANCE type already exists. So this most likley needs to be merged into that type.